### ISSUE TITLE
403/ERR_CONNECTION_REFUSED after fresh install on Debian - SSL Issue?

### ISSUE THREAD

**Issue by zamora636** [SeafileDE/seafile-server-installer#84]
After digging through the logs I couldn't see any issues during the install/script run.

That's when I decided to check nginx:

root@i:~# service nginx restart
Restarting nginx: nginx: [emerg] SSL_CTX_use_certificate_chain_file("/etc/nginx/ssl/seafile.crt") failed (SSL: error:02001002:system library:fopen:No such file or directory error:20074002:BIO routines:FILE_CTRL:system lib error:140DC002:SSL routines:SSL_CTX_use_certificate_chain_file:system lib)
nginx: configuration file /etc/nginx/nginx.conf test failed

It seems to be an SSL issue.

Any assistance is appreciated.

**Comment by zamora636** [comment]
TLDR - Fixed it by manually remaking the SSLs.

Long version:

Tried to remake SSLs, since I noticed no .pem or .crt working, only 0 byte files.

First I ran:

root@i:/etc/nginx# openssl dhparam -dsaparam -out /etc/nginx/dh4096.pem 4096

Which worked fine.

Then this command gave me a new error:

root@i:/etc/nginx# openssl req -new -x509 -key /etc/nginx/ssl/seafile.key -out /etc/nginx/ssl/seafile.crt -days 10950 -config /etc/nginx/ssl/seafile.cnf -sha256
Error Loading extension section v3_ca
140118112302760:error:220A4076:X509 V3 routines:A2I_GENERAL_NAME:bad ip address:v3_alt.c:477:value=::1 127.0.0.1
140118112302760:error:22098080:X509 V3 routines:X509V3_EXT_nconf:error in extension:v3_conf.c:93:name=subjectAltName, value=@alt_names

What I then did was edit the .cnf file (think Cert request) and changed the IPs at the bottom from 127.0.01 to the IP of my VPS. After savings those changes I was then able to run the openssl generate command successfully.

Steps:

root@i:/etc/nginx# nano /etc/nginx/ssl/seafile.cnf

Change the IPs at the bottom from 127.0.01 to your VPS IP(s).

Then save and lastly edit your nginx config at "/etc/nginx/conf.d/seafile.conf" and ensure it points to your new crt file. 

Lastly restart nginx with:

service nginx restart.

**Comment by alexanderjackson** [comment]
Seems like your host name wasn't set correctly before running the installer. I'm closing the issue.
